iroh-bitswap/src/metrics.rs

```diff
@@ -1,9 +1,15 @@
-use git_version::git_version;
 use iroh_metrics::config::Config as MetricsConfig;
 
 pub fn metrics_config_with_compile_time_info(cfg: MetricsConfig) -> MetricsConfig {
     // compile time configuration
     cfg.with_service_name(env!("CARGO_PKG_NAME").to_string())
-        .with_build(git_version!().to_string())
+        .with_build(
+            git_version::git_version!(
+                prefix = "git:",
+                cargo_prefix = "cargo:",
+                fallback = "unknown version"
```

Comment:
for CLI sake I'd prefer fallback to be non space separated or just `unknown`

Reply:
fixed